cli/command/container/unpause.go

```diff
@@ -27,6 +29,9 @@ func NewUnpauseCommand(dockerCli command.Cli) *cobra.Command {
 			opts.containers = args
 			return runUnpause(dockerCli, &opts)
 		},
+		ValidArgsFunction: completion.CompleteContainerNames(dockerCli, false, func(container types.Container) bool {
+			return container.Status != "pause"
```

Comment:
> So we should move the human-readable translation in the CLI and deprecate this ASAP

Yeah, it's slightly complicated; the field can include (as shown above) `Up XX minutes`. For the CLI to show the correct duration (there's a ticket for that somewhere; I can try find it), we need it to know about the server time. So we either have to include that in some endpoints, or (alternatively) could return a Header in all responses that can be used by the CLI to do these calculations.

Another example is the `Created` (I think?) field, which is currently changed into a "human friendly" relative time, but this may show the wrong "XX minutes ago" if the daemon and cli's clocks are out of sync.

Reply:
Maybe the server should return a duration instead of a date/time.